tardis/base.py

```diff
@@ -16,27 +16,39 @@ def run_tardis(
     specific=None,
 ):
     """
-    This function is one of the core functions to run TARDIS from a given
-    config object.
+    Run TARDIS from a given config object.
 
-    It will return a model object containing
+    It will return a model object containing the TARDIS Simulation.
 
     Parameters
     ----------
     config : str or dict or tardis.io.config_reader.Configuration
         filename of configuration yaml file or dictionary or TARDIS Configuration object
-    atom_data : str or tardis.atomic.AtomData
-        if atom_data is a string it is interpreted as a path to a file storing
+    atom_data : str or tardis.atomic.AtomData, optional
+        If atom_data is a string it is interpreted as a path to a file storing
         the atomic data. Atomic data to use for this TARDIS simulation. If set to None, the
         atomic data will be loaded according to keywords set in the configuration
-        [default=None]
-    virtual_packet_logging : bool
-        option to enable virtual packet logging
-        [default=False]
+    packet_source : class, optional
+        A custom packet source class or a child class of `tardis.montecarlo.packet_source`
+        used to override the TARDIS `BasePacketSource` class.
+    simulation_callbacks : list of lists, optional
+        Set of callbacks to call at the end of every iteration of the Simulation.
+        The format of the lists should look like:
+        [[callback1, callback_arg1], [callback2, callback_arg2], ...]
+        Where the callback function signature should look like:
+        callback_function(simulation, extra_arg1, ...)
+    virtual_packet_logging : bool, default False, optional
+        Option to enable virtual packet logging.
+    log_state : str, optional
+        Set the level of the TARDIS logger. Options include `NOTSET`, `DEBUG`, `INFO`, `WARNING`, `ERROR`, and `CRITICAL`.
+        Default level of the logger is set to CRITICAL.
+    specific : bool, optional
```

Comment:
See this - you are also breaking consistency by mentioning some default values in parameter type - let's keep all in description

Reply:
The default value of the argument in the function is `None`, but that of the logger is actually `CRITICAL`, which is set [here](https://github.com/tardis-sn/tardis/blob/de2d7be6dbafac82ab538bdf13526c48a1cbf10e/tardis/io/logger/logger.py#L32) I believe. I guess moving this to the parameter section would mean that the default argument is `CRITICAL`, but it's actually `None`.

Reply:
Then add default: None here and explain in description what does none imply